sklearn/model_selection/_validation.py

```diff
@@ -781,6 +781,23 @@ def _score(estimator, X_test, y_test, scorer, error_score="raise"):
                 UserWarning,
             )
 
+    # Check errors in `_MultimetricScorer`
+    if isinstance(scorer, _MultimetricScorer):
+        exceptions = {name: e for name, e in scores.items() if isinstance(e, Exception)}
+        if exceptions:
+            if error_score == "raise":
+                for name, e in exceptions.items():
+                    raise e
+            else:
+                new_scores = {name: error_score for name in exceptions}
+                scores.update(new_scores)
+                warnings.warn(
+                    "Scoring failed. The score on this train-test partition for "
+                    f"these parameters will be set to {error_score}. Details: \n"
+                    f"{format_exc()}",
```

Comment:
I think we can not use `format_exc` here since this is not in a `except` block. We would need to look at `exceptions` for the information.

Reply:
How can we show here the content of `exceptions`? I think it would make sence to show every error message here, not just the first one.

Reply:
This way maybe, but with formatted `e`?

```
else:  # line 772
    for name, e in exceptions:
        scores[name] = error_score
        warnings.warn(
            "Scoring failed. The score on this train-test partition for "
            f"these parameters will be set to {error_score}. Details: \n"
            f"{e}",  # TODO: format like in case of `format_exc()`
            UserWarning,
        )
```

Reply:
> How can we show here the content of exceptions? I think it would make sence to show every error message here, not just the first one.

Yup. I think it makes sense to show all of them.